react/components/PickUpHeader.tsx

```diff
@@ -1,22 +1,24 @@
-import React, { FunctionComponent } from 'react'
+import React, { FC } from 'react'
 import { FormattedMessage } from 'react-intl'
 import { TranslateEstimate } from 'vtex.shipping-estimate-translator'
 import { Address } from 'vtex.order-details'
 
-const StorePickUpHeader: FunctionComponent<Props> = ({
-  shippingData,
-  index,
-  numPackages,
-}) => {
+interface Props {
+  shippingData: Parcel
```

Comment:
where does this kind of `parcel` come from?

Reply:
`global.d.ts` file: https://github.com/vtex-apps/order-placed/blob/master/react/typings/global.d.ts#L153